code/modules/elevator/elevator_segment.dm

```diff
@@ -8,6 +8,9 @@
 	var/turf/base_turf
 	///List of things we refuse to transport
 	var/static/list/move_blacklist = typecacheof(list(/atom/movable/lighting_object, /obj/structure/cable, /obj/structure/disposalpipe, /obj/machinery/atmospherics/pipe))
+	///List of elevator music files
+	var/list/music_files = list('sound/effects/turbolift/elevatormusic.ogg', 'sound/effects/turbolift/elevator_loop.ogg')
```

Comment:
one last thing, initializing typecaches and lists or empty lists outside of `Initialize` is bad for performance, you should move all list assignments and typecache inits to that. same applies to the other new code. subsystems I think are an exception case